src/edu/washington/escience/myria/operator/CSVFileScanFragment.java

```diff
@@ -430,6 +441,23 @@ protected void init(final ImmutableMap<String, Object> execEnvVars) throws DbExc
                 new BufferedReader(new InputStreamReader(partitionInputStream)),
                 CSVFormat.newFormat(delimiter).withQuote(quote).withEscape(escape));
         iterator = parser.iterator();
+
+        try {
+          if (!iterator.hasNext()) {
```

Comment:
So I changed this to ```adjustedStartByteRange + byteOffsetFromTruncatedRowAtStart -1 == partitionEndByteRange```

-1 because the ```byteOffsetFromTruncatedRowAtStart``` is counting the bytes inclusively based on the byte range. For example if the ```adjustedStartByteRange``` is 3 and the ```partitionEndByteRange``` is 7, then the ```byteOffsetFromTruncatedRowAtStart``` would be 5.